docs/src/developing/clients/jsonrpc-api.md

```diff
@@ -2074,6 +2079,7 @@ An array of:
 - `RpcPerfSample<object>`
   - `slot: <u64>` - Slot in which sample was taken at
   - `numTransactions: <u64>` - Number of transactions in sample
+  - `numNonVoteTransaction: <u64>` - Number of non-vote transactions in sample
```

Comment:
Now I wonder if I should clarify ``numNonVoteTransaction` with an explanation that it is only present after upgrade to v1.15 and only for newer blocks.

Reply:
Yeah, I was going to say that if we do document the version added, the public JSON-RPC docs are probably the place to do it. Historically, we've only done that for input parameters, not new output fields. But we could start.

Reply:
We do have some precedent of doing so for new methods:
https://github.com/solana-labs/solana/blob/055c6a5ebd06352d69cf3cebeee882c3ef9c527c/docs/src/developing/clients/jsonrpc-api.md?plain=1#L1056